tests/test_base.py

```diff
@@ -399,16 +399,7 @@ def test_default_if_none_with_none():
 
 
 FALSEY_VALUES = [None, "", [], {}, False, 0]
-SINGLE_ITEM_VALUES = [
-    1,
-    11111,
-    "1",
-    "ABCDEFG",
-    u"ABCDEFG",
-    [[]],
-    {"A": 1, "B": 2},
-    True,
-]
+SINGLE_ITEM_VALUES = [1, 11111, "1", "ABCDEFG", "ABCDEFG", [[]], {"A": 1, "B": 2}, True]
```

Comment:
This was "black" doing its thang.

Reply:
I know this is pot-merge, so it's just an observation, but we now have two identical `"ABCDEFG"` strings since `black` removed the `u` on the second one. Being of unsound mind and body, should that be a `b` instead since Unicode no longer has any distinct meaning in Py3?

Reply:
One does not question Black!
I simply obeyed it's change as I did not modify this code, I guess a newer version of black wanted things cleaned up more?